compiler/rustc_mir/src/interpret/step.rs

```diff
@@ -113,6 +113,49 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
                 M::retag(self, *kind, dest)?;
             }
 
+            // Call CopyNonOverlapping
+            CopyNonOverlapping(box rustc_middle::mir::CopyNonOverlapping { dst, src, count }) => {
```

Comment:
Looks like you can copy https://github.com/rust-lang/rust/blob/7a7dcc85fd3da873b6b64f103abf1bb8e41268ab/compiler/rustc_mir/src/interpret/intrinsics.rs#L327-L349